Configuration/Eras/python/Era_Run2_2017_cff.py

```diff
@@ -21,8 +21,10 @@
 from Configuration.Eras.Modifier_pixel_2016_cff import pixel_2016
 from Configuration.Eras.Modifier_run2_jme_2017_cff import run2_jme_2017
 from Configuration.Eras.Modifier_run2_jme_2016_cff import run2_jme_2016
+from Configuration.Eras.Modifier_strips_vfp30_2016_cff import strips_vfp30_2016
 
-Run2_2017 = cms.ModifierChain(Run2_2016.copyAndExclude([run2_muon_2016, run2_HLTconditions_2016,run2_egamma_2016,run2_tau_ul_2016,pixel_2016,run2_jme_2016]),
+
+Run2_2017 = cms.ModifierChain(Run2_2016.copyAndExclude([run2_muon_2016, run2_HLTconditions_2016,run2_egamma_2016,run2_tau_ul_2016,pixel_2016,run2_jme_2016, strips_vfp30_2016]),
```

Comment:
a new empty line could be dropped.
Since the autoCond update is still expected, please fix here then.

Reply:
sure, I'll make the update once the new GT is available.

Reply:
it is done in 366b3e7eb1a8ee25807dce472b51d4ea17ac8950